docs/usage.md

```diff
@@ -88,6 +95,31 @@ Sample files that can be used as illustrative references can be found in this re
 | last_name | str | User's last name |
 | email | str | User's email |
 
+## `quiz_questions.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| assessment_id | int | A unique assessment id |
+| question_number | int | The relative order number of the question  |
+| question_id | UUID | A unique identifier for the question number |
+
+## `quiz_question_contents.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | The unique question id |
+| text | str | The question's text |
+| type | ['multichoice', 'multianswer', 'numerical', 'essay'] | The type of question |
+
+## `quiz_multichoice_answer.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | int | A unique assessment id |
+| question_id | UUID | A unique identifier for the question number |
+| text | str | The answer text |
+| grade | float | The percentage of the questions total points received for answering this answer |
```

Comment:
Minor editorial nit: Change to "The percentage of the question's total points received for this answer"

Reply:
@mjoss17 - Just nudging this comment (minor editorial).